generators/app/index.js

```diff
@@ -126,7 +126,7 @@ function showPrompts() {
         imageName = props.imageName;
         serviceName = props.serviceName;
         baseImageName = props.baseImageName;
-        composeProjectName = props.composeProjectName;
+        composeProjectName = props.composeProjectName.replace(/[^a-zA-Z0-9]/g, '');
```

Comment:
>/[^a-zA-Z0-9]/g [](start = 62, length = 15)

Does this include _ (underscore) ? I guess that is also not allowed in project name, right?